pandas/core/resample.py

```diff
@@ -895,6 +895,74 @@ def asfreq(self, fill_value=None):
         """
         return self._upsample("asfreq", fill_value=fill_value)
 
+    def sum(
```

Comment:
no docstring to inherit?

Reply:
For these one that were left out, I couldn't inherit since the associated groupby doc contains numba arguments that are not supported by resample.